src/libs/SignatureChecker.sol

```diff
@@ -44,6 +63,11 @@ library SignatureChecker {
         bytes32 digest,
         bytes memory signature
     ) internal view returns (bool isValid) {
+        if (signature.length == 64) {
+            (bytes32 r, bytes32 vs) = decodeShortECDSASignature(signature);
+            return isValidECDSASignature(signer, digest, r, vs);
+        }
```

Comment:
This will be a bug, since a 64 byte `ERC1271` signature will fail the  `ECDSASignature` check. Should also create regression test case for this.

Reply:
true. Can this check live inside `isValidECDSASignature` ?

Reply:
Fixed in this commit: https://github.com/MZero-Labs/common/pull/8/commits/1ee530c43bf98aad3dc10d5417628f2f23db63d7